src/json/members/rydersit.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Ryder Sitcawich",
+    "pennkey": "rydersit",
+    "major": "Computer Science",
+    "school": "Engineering",
+    "bio": "Hey, I'm Ryder and I'm a Sophomore from Minnesota! In my free time I enjoy playing pickup soccer, cooking beef wellington, and losing my money in poker.",
+    "hometown": "Woodbury, MN",
+    "team": "Penn Courses",
+    "roles": ["Frontend"],
+    "photo": "https://imgur.com/aM2cvav",
```

Comment:
This is a website url, not an image URL. You need to right click on the photo and select "Copy Image Address".